gen/definitions/segment_routing_isis.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
We already have a resource `iosxr_router_isis_address_family` so we should add the missing attributes there.

Reply:
checking

Reply:
done